JavaScript/JavaScriptSDK.Tests/CheckinTests/Sender.tests.ts

```diff
@@ -1111,6 +1111,10 @@ class SenderTests extends TestClass {
         this.testCase({
             name: "SenderTests: send() is using BeaconAPI sender if the BeaconAPI is enabled",
             test: () => {
+                if (!(<any>navigator).sendBeacon) {
```

Comment:
I think I added that when I was fixing unit tests and forgot to remove it. It's gone now.